app/assets/javascripts/_init-autocomplete.js

```diff
@@ -1,9 +1,18 @@
 jQuery.fn.extend({
   geonameAutocomplete: function () {
     return this.autocomplete({
-      minLength: 2,
-      source: function (request, response) {
-        $.getJSON("/cities/autocomplete?term=" + request.term, function (data) {
+      minLength: 0,
+      // automatically focus first item from autocomplete menu
+      autoFocus: true,
+      source: function(request, response) {
+        var listUri;
+        if (request.term.length < 2) {
```

Comment:
I don't think it should be implemented like that or it needs further work. When I type "P" I get Paris (nice but it's showing two times ??!!). Problem is that when I type "Paris" the first selected item reverses to Parisot. It's really troubling.
![citiy complete](https://user-images.githubusercontent.com/540760/31854090-60a416e2-b694-11e7-8ff9-ef1373469417.png)
 Maybe we should put an exact match always first ?

Reply:
Sorry, I added cities multiple times on test env, fixe.

Reply:
What I meant: For example on Paris:
Typing "P" gives Paris, but typing "Pa" gives Parisot and further letters won't change the order (typing "Paris" gives Parisot).
I think we should boost main cities even when more than one letter is entered and especially if we have an exact match with the name.

Reply:
Ok good idea, check or create an issue for it, linked to suggested cities.